test cases/d/11 dub/meson.build

```diff
@@ -1,12 +1,15 @@
 project('dub-example', 'd')
 
-error('MESON_SKIP_TEST: Dub support is broken at the moment (#11773)')
```

Comment:
Hold on, why was this ever unconditionally errored out in the first place???

We should *never* do that. Dub support wasn't broken until after the version is checked for the version that introduced breakage.

Disabling tests on *all* versions of Dub means we get zero integration tests for anything at all, even the commits that introduced conditional errors.